src/styles/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+}
+
+.search-bars {
+  width: 100%;
+}
+
+.search-bar {
+  display: flex;
+  margin-inline: 8px;
+  align-items: center;
+}
+
+.search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+}
+
+.search-bar__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  text-shadow: 0 4px 4px #00000040;
+  font-weight: 900;
+  outline: none;
+}
+
+.search-bar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  line-height: 21.86px;
```

Comment:
Looks strange, if you have problems with text align it's better to use vertical paddings in this case.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/108701488/b3fac69b-3e2c-41e1-936c-78599ec4afd1)

Reply:
the same is for small input